tests/ssl-opt.sh

```diff
@@ -4733,8 +4735,162 @@ run_test    "Record Size Limit: TLS 1.3: Client-side parsing, debug output and f
 #            -s "Sending extension Record Size Limit/28 (2 bytes)" \
 #            -c "EncryptedExtensions: record_size_limit(28) extension received."\
 #            -c "found record_size_limit extension" \
-#            -c "RecordSizeLimit: 16385 Bytes" \
-#            -s "Received alert \[110]: An unsupported extension was sent"
+#            -c "RecordSizeLimit: 16385 Bytes"
+
+# In the following (9) tests, --recordsize is the value used by the G_NEXT_CLI (3.7.2) to configure the
```

Comment:
How about the test cases with the peer from OpenSSL.

Reply:
OpenSSL does not support record size limit yet